.github/workflows/ci.yml

```diff
@@ -115,9 +224,152 @@ jobs:
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+          RUN_ID: ${{ github.run_id }}
+          COMMIT_SHA: ${{ github.event.pull_request.head.sha }}
+        run: |
+          if [ "$GITHUB_EVENT_NAME" == pull_request ]; then
+            WORKFLOW_NAME=publish-test-reports-pr.yml
+            gh workflow run publish-test-reports-pr.yml \
+              -f run_id=$RUN_ID \
+              -f e2e_artifact=e2e-test-report \
+              -f pr_number=$PR_NUMBER \
+              -f commit_sha=$COMMIT_SHA \
+              -f s3_root=public \
+              --repo woocommerce/woocommerce-test-reports
+          else
+            gh workflow run publish-test-reports-trunk-merge.yml \
+              -f run_id=$RUN_ID \
+              -f artifact=e2e-test-report \
+              -f pr_number='' \
+              -f test_type="e2e" \
+              --repo woocommerce/woocommerce-test-reports
+          fi
+
+
+  slack-alert-e2e-test:
+    name: 'Send Slack alert for e2e tests'
+    needs: [ project-e2e-test-jobs ]
+    if: ${{ github.event_name != 'pull_request' && ( contains( needs.*.result, 'success' ) || contains( needs.*.result, 'failure' ) ) }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: 'Send Slack notification'
+        uses: automattic/action-test-results-to-slack@v0.3.0
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          slack_token: ${{ secrets.E2E_SLACK_TOKEN }}
+          slack_channel: ${{ secrets.E2E_TRUNK_SLACK_CHANNEL }}
+
+  publish-api-test-reports:
+    name: 'Publish API test reports'
+    needs: [ project-api-test-jobs ]
+    if: ${{ contains( needs.*.result, 'success' ) || contains( needs.*.result, 'failure' ) }}
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: 'Install Allure CLI'
+        env:
+          DESTINATION_PATH: ../
+        run: ./.github/workflows/scripts/install-allure.sh
+
+      - name: 'Download blob reports from artifacts'
+        uses: actions/download-artifact@v4
+        with:
+          path: ./out/allure-results
+          pattern: all-blob-reports-*
+          run-id: project-api-test-jobs
+          merge-multiple: true
+
+      - name: 'Generate Allure report'
+        id: generate_allure_report
+        run: allure generate --clean ./out/allure-results --output ./out/allure-report
+
+      - name: 'Archive reports'
+        uses: actions/upload-artifact@v4
+        with:
+          name: api-test-report
+          path: ./out
+          if-no-files-found: ignore
+          retention-days: 5
+          
+      - name: 'Publish reports'
+        env:
+          GITHUB_TOKEN: ${{ secrets.REPORTS_TOKEN }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+          RUN_ID: ${{ github.run_id }}
+          COMMIT_SHA: ${{ github.event.pull_request.head.sha }}
+        run: |
+          if [ "$GITHUB_EVENT_NAME" == pull_request ]; then
+            WORKFLOW_NAME=publish-test-reports-pr.yml
```

Comment:
Is `WORKFLOW_NAME` used anywhere?

Reply:
Nice catch! It doesn't seem to be used anywhere, it must have been left over the copy-paste and the small adjustments I had to make to not break the report publishing.
Removed with [`64e4a04` (#45190)](https://github.com/woocommerce/woocommerce/pull/45190/commits/64e4a041d06d99a05526445a6e656808d8b236da), and reporting seems to still work.